sycl/doc/GetStartedWithSYCLCompiler.md

```diff
@@ -390,6 +416,18 @@ clang++ -fsycl -fsycl-targets=nvptx64-nvidia-cuda-sycldevice \
 This `simple-sycl-app.exe` application doesn't specify SYCL device for
 execution, so SYCL runtime will use `default_selector` logic to select one
 of accelerators available in the system or SYCL host device.
+In this case, the behaviour of the `default_selector` can be altered 
+using the `SYCL_BE` environment variable, setting `PI_CUDA` forces
+the usage of the CUDA backend (if available), `PI_OPENCL` will
+force the usage of the OpenCL backend.
+
+```bash
+SYCL_BE=PI_CUDA ./simple-sycl-app-cuda.exe
+```
+
+The default is the OpenCL backend if available.
+
+
 
 Note: `nvptx64-nvidia-cuda-sycldevice` is usable with `-fsycl-targets`
```

Comment:
`nvptx64-unknown-unknown-sycldevice` or ``nvptx64-*-*-sycldevice` seem to work as well (basically the 2nd and 3rd argument of the triple can be anything, or even left out as in `nvptx64---sycldevice`).

Reply:
There is also another triple, `nvptx64-nvidia-opencl-sycldevice`, that can be selected. Its unclear to me which one comes by default. 
From what I recall of the initial investigation (this was a while ago), the `-opencl-` one generates a plain PTX file whereas the `-cuda-` one generates a cubin, @Alexander-Johnston  / @Naghasan  will know better.
The `-cuda-` variant is the one we use on the lit tests, so its the one I suggest people to use.